src/Core/Maintenance/System/Command/SystemInstallCommand.php

```diff
@@ -40,6 +40,7 @@ protected function configure(): void
             ->addOption('shop-email', null, InputOption::VALUE_REQUIRED, 'Shop email address')
             ->addOption('shop-locale', null, InputOption::VALUE_REQUIRED, 'Default language locale of the shop')
             ->addOption('shop-currency', null, InputOption::VALUE_REQUIRED, 'Iso code for the default currency of the shop')
+            ->addOption('skip-jwt-keys-generation', null, InputOption::VALUE_NONE, 'Skips generation of jwt private and public key')
```

Comment:
not sure about the name and also not sure if it should be skip or generate. but if I use generate then it changes the commands default behavior and would be breaking.